### PR TITLE
docs: fix doc

### PR DESCRIPTION
**Pull request by Sherlock-Holo**
fix doc

### CODE REVIEW

File: tencentcloud/resource_tc_gaap_security_rule.go [hunk 1]
```diff
@@ -105,7 +105,7 @@ func resourceTencentCloudGaapSecurityRule() *schema.Resource {
 					}
 					return
 				},
-				Description: "Target port. Available values includes `80`,`80,443`,`3306-20000`.",
+				Description: "Target port. Default is `ALL`, the available values includes `80`,`80,443`,`3306-20000`.",
```
Comment (gailwang) [inline tencentcloud/resource_tc_gaap_security_rule.go]: inlcudes后面，没有空格

Comment (gailwang) [reply tencentcloud/resource_tc_gaap_security_rule.go]: includes --> include

Comment (likexian) [reply tencentcloud/resource_tc_gaap_security_rule.go]: Possible values include

Comment (Sherlock-Holo) [reply tencentcloud/resource_tc_gaap_security_rule.go]: 用 `the available values` 吧，看很多 resource 都是用的这个

File: tencentcloud/data_source_tc_gaap_proxies.go [hunk 2]
```diff
@@ -38,7 +38,7 @@ func dataSourceTencentCloudGaapProxies() *schema.Resource {
 				Optional:      true,
 				Elem:          &schema.Schema{Type: schema.TypeString},
 				ConflictsWith: []string{"project_id", "access_region", "realserver_region"},
-				Description:   "ID of the GAAP proxy to be queried. Conflict with `project_id`,`access_region`,`realserver_region`.",
+				Description:   "ID of the GAAP proxy to be queried. Conflict with `project_id`, `access_region` amd `realserver_region`.",
```
Comment (likexian) [inline tencentcloud/data_source_tc_gaap_proxies.go]: amd -> and

File: website/docs/r/gaap_security_rule.html.markdown [hunk 3]
```diff
@@ -38,12 +38,12 @@ resource "tencentcloud_gaap_security_rule" "foo" {
...
 
-* `action` - (Required, ForceNew) Policy of the rule, the available values includes `ACCEPT` and `DROP`.
+* `action` - (Required, ForceNew) Policy of the rule, the available values include `ACCEPT` and `DROP`.
 * `cidr_ip` - (Required, ForceNew) A network address block of the request source.
 * `policy_id` - (Required, ForceNew) ID of the security policy.
 * `name` - (Optional) Name of the security policy rule. Maximum length is 30.
-* `port` - (Optional, ForceNew) Target port. Available values includes `80`,`80,443`,`3306-20000`.
-* `protocol` - (Optional, ForceNew) Protocol of the security policy rule. Default is `ALL`, the available values includes `TCP`,`UDP` and `ALL`.
+* `port` - (Optional, ForceNew) Target port. Default value is `ALL`, the available values include `80`, `80,443` and `3306-20000`.
+* `protocol` - (Optional, ForceNew) Protocol of the security policy rule. Default value is `ALL`, the available values include `TCP`, `UDP` and `ALL`.
```
Comment (likexian) [inline website/docs/r/gaap_security_rule.html.markdown]: The available values
和
Available values

建议统一，好几个地方

File: tencentcloud/data_source_tc_gaap_realservers.go [hunk 4]
```diff
@@ -33,7 +33,7 @@ func dataSourceTencentCloudGaapRealservers() *schema.Resource {
 			"project_id": {
 				Type:        schema.TypeInt,
 				Optional:    true,
-				Description: "ID of the project within the GAAP realserver to be queried, default is '-1' means all projects.",
+				Description: "ID of the project within the GAAP realserver to be queried, default is `-1` means all projects.",
```
Comment (Himer) [inline tencentcloud/data_source_tc_gaap_realservers.go]: default:-1

no set means all projects

**Comment by Sherlock-Holo** [comment]
## data source

> === RUN   TestAccDataSourceTencentCloudGaapRealservers_basic
> --- PASS: TestAccDataSourceTencentCloudGaapRealservers_basic (3.29s)
> === RUN   TestAccDataSourceTencentCloudGaapRealservers_domain
> --- PASS: TestAccDataSourceTencentCloudGaapRealservers_domain (1.44s)
> === RUN   TestAccDataSourceTencentCloudGaapRealservers_name
> --- PASS: TestAccDataSourceTencentCloudGaapRealservers_name (1.42s)
> PASS
> ok      github.com/terraform-providers/terraform-provider-tencentcloud/tencentcloud     6.195s

**Pull request merged**